web/bfex/components/data_pipeline/tasks/persist.py

```diff
@@ -51,6 +51,9 @@ def run(self, data):
             if "researchid_link" in scrapp.meta_data:
                 faculty.research_id = scrapp.meta_data["researchid_link"]
 
+            if "text" in scrapp.meta_data:
+                 faculty.text = scrapp.meta_data["text"]
```

Comment:
I'm not sure if we want to store this data inside the faculty member document. It makes sense in that it is obviously linked to a single person, and fits in better with our current workflow. But I also think it'd be nice to have all documents in a seperate index. There are certain operations (ie. tf-idf) that we may not want to run on everything in the faculty index.

Let's revisit this during our next meeting.